unstructured/ingest/cli/cmds/chroma.py

```diff
@@ -27,17 +27,15 @@ def get_cli_options() -> t.List[click.Option]:
             ),
             click.Option(
                 ["--tenant"],
-                required=False,
-                default="default_tenant",
+                required=True,
                 type=str,
-                help="The tenant to use for this client.",
+                help="The tenant to use for this client." "Chroma default is 'default_tenant'.",
             ),
             click.Option(
                 ["--database"],
-                required=False,
-                default="default_database",
+                required=True,
                 type=str,
-                help="The database to use for this client.",
+                help="The database to use for this client." "Chroma default is 'default_database'.",
```

Comment:
sorry, circling back from DMs. did not realize Chroma actually defaults to these values itself. in which case this is fine! ...maybe add a comment somewhere to that extent